### ISSUE TITLE
`faiss::FaissException` when using `vss_search` with `better-sqlite3` on Node (WSL)

### ISSUE THREAD

**Issue by larryhudson** [asg017/sqlite-vss#81]
Hi Alex, awesome job with these sqlite extensions. I'm trying to get up and running with `sqlite-vss` and `better-sqlite3` on Node, in Ubuntu using WSL.

I have followed the installation instructions and have set up a simple test script here:
https://github.com/larryhudson/testing-sqlite-vss/blob/main/index.js

I think I am storing the vectors and setting up the virtual table properly, but I am running into a `faiss::FaissException` error when I get to the `vss_search` stage:
```sh
terminate called after throwing an instance of 'faiss::FaissException'
  what():  Error in virtual void faiss::IndexFlat::search(faiss::idx_t, const float*, faiss::idx_t, float*, faiss::idx_t*, const faiss::SearchParameters*) const at /home/runner/work/sqlite-vss/sqlite-vss/vendor/faiss/faiss/IndexFlat.cpp:34: Error: 'k > 0' failed
Aborted
```

Could you please check out the test script and let me know if I am doing something wrong? Thanks again.

**Comment by larryhudson** [comment]
Just tried reproducing the error on macOS (M1) and I'm getting the same error, so I don't think it's a WSL error. Must be something wrong with how I'm setting up the tables or storing the vectors.

**Comment by larryhudson** [comment]
Ah great, that has fixed it! I missed that instruction when I was going through the Observable article.

Thanks for the quick response :)

**Comment by larryhudson** [comment]
And it looks like I'm running SQLite version 3.39.5 on macOS, by the way.

**Comment by asg017** [comment]
Glad to hear it got resolved!

One note, since you're on `3.39.5` (which contains a bug that was fixed in `3.41+`, you'll have to use the `vss_search(text_embedding, vss_search_params(?, 20))` syntax instead of `vss_search(text_embedding, ?) limit 20`. There's some info about this in the docs, which I should also clean up a little...
